capstone/fitness/src/main/java/com/google/sps/progress/Progress.java

```diff
@@ -0,0 +1,200 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.progress;
+
+import com.google.sps.fit.*;
+import com.google.sps.fit.FitnessSet.SetType;
+import com.google.sps.util.*;
+import java.util.*;
+
+/* Handler for ProgressModel updates. */
```

Comment:
Can you add more detailed descriptions? It's quite hard to understand what this class and its methods do.